### ISSUE TITLE
A Bot That Writes Signed Books

### ISSUE THREAD

**Issue by Gastlt** [ImpactDevelopment/ImpactClient#706]
*Suggestion*

It would be pretty nice to have a module that gets a text file from your documents and writes all of it into minecraft signed books.

**Comment by SuperOP535** [comment]
Can't you just paste?

**Comment by Gastlt** [comment]
you cant

**Comment by Ilidumb** [comment]
A neat idea, would be pretty usefull!

**Comment by 0-x-2-2** [comment]
you can copypaste what are you talking about

**Comment by VitorISs** [comment]
@0-x-2-2 you actually can't copy paste

**Comment by 0-x-2-2** [comment]
uhh yeah you can?

**Comment by Mhowser** [comment]
i can copy and paste the "§" symbol for formatted text.

**Comment by Thunder33345** [comment]
well would that still do for multipaged books?
signing the book still aint auto

**Comment by LeafHacker** [comment]
Just because there are alternatives available doesn't _automatically_ invalidate a feature request. It does lesson its priority though.

**Comment by Mhowser** [comment]
You are correct, I too would like this as a feature.

On Oct 10, 2018 3:34 PM, "Leaf" <notifications@github.com> wrote:

Just because there are alternatives available doesn't *automatically*
invalidate a feature request. It does lesson its priority though.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/ImpactDevelopment/ImpactClient/issues/706#issuecomment-428756274>,
or mute the thread

**Comment by SuperOP535** [comment]
I don't think this will be needed once Impact is 1.14 (in about 1-2 years) because there are some nice book- and sign-editing stuff.

**Comment by 0-x-2-2** [comment]
1.14 is in early 2019 aka few months not year(s)...

**Comment by SuperOP535** [comment]
Never said MC would take 1-2 years, I said Impact will probably take 1-2 years to update to 1.14 📦

**Comment by Thunder33345** [comment]
singing still kinda needed and you cant just let MC help you copywrite it without actively doing it, rather then begin able to AFK and let the bot do it's things

**Comment by LeafHacker** [comment]
The vanilla sign/book GUI improvements could probably be backported to 1.12 impact now, but if that is desired you should open a separate issue.

My understanding of this issue is that writing _and_ signing books should be fully AFK-able (until you run out of books, at least).

**Comment by sapioit** [comment]
New issue continuing this one: #1066

**Comment by Michael18751** [comment]
Yes! Just like Forgehax!

**Comment by Michael18751** [comment]
This also would be useful for dupping
